presto-tests/src/main/java/io/prestosql/tests/H2QueryRunner.java

```diff
@@ -95,7 +96,7 @@
 
     public H2QueryRunner()
     {
-        handle = Jdbi.open("jdbc:h2:mem:test" + System.nanoTime());
+        handle = Jdbi.open("jdbc:h2:mem:test" + ThreadLocalRandom.current().nextLong());
```

Comment:
This can, in theory, still produce repeated values. You may want to combine both or use a global counter (AtomicLong), instead.

Reply:
`jdbc:h2:mem:` occurs 26 time in the code, most often as `"jdbc:h2:mem:test" + System.nanoTime()`. I believe you want to fix them all the same way.